### PR TITLE
Use the "latest" VM images

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -127,13 +127,7 @@ stages:
...
       - job: Linux
         displayName: 'Linux'
-        pool:
-          ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
-            name: NetCore1ESPool-Public
-            demands: ImageOverride -equals Build.Ubuntu.1604.Amd64.Open
-          ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-            name: NetCore1ESPool-Internal
-            demands: ImageOverride -equals Build.Ubuntu.1604.Amd64
+        pool: 'ubuntu-latest'
```
Comment (MattGal) [inline azure-pipelines.yml:130]: needs "vmimage".  See https://docs.microsoft.com/en-us/azure/devops/pipelines/agents/hosted?view=azure-devops&tabs=yaml#software

Comment (tmat) [reply azure-pipelines.yml]: Updated

File: azure-pipelines.yml [hunk 2]
```diff
@@ -48,12 +48,7 @@ stages:
       jobs:
       - job: Windows
         pool:
-          ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
-            name: NetCore1ESPool-Public
-            demands: ImageOverride -equals Build.Windows.10.Amd64.VS2017.Open
-          ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
-            name: NetCore1ESPool-Internal
-            demands: ImageOverride -equals Build.Windows.10.Amd64.VS2017
+          vmImage: 'windows-latest'
```
Comment (riarenas) [inline azure-pipelines.yml:51]: I see that the sourcelink official build is doing signing, which won't work in the hosted pools. You should probably keep the ifs, and only use hosted for the public CI and PR builds.

Comment (tmat) [reply azure-pipelines.yml:51]: I see.

**Pull request closed without merging**